SU-HSDO/stanford_mrc
# Mrcd8 252 media fields

# READY FOR REVIEW

# Summary
- A WHOLE LOTTA CHANGES:
- Created new media fields on entity types
- migrate image/file/video fields to the new media fields
- new field formatter for entity_reference Media fields
- Entity display modes updated to use the new media fields.

# Needed By (Date)
- End of sprint

# Urgency
- High

# Steps to Test

1. Checkout this branch
2. sync to prod
3. update database
4. view & edit different nodes
5. verify the nodes still look basically the same
6. verify the edit page has the applicable media browser

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
